docs/root/start/sandboxes/_include/dynamic-configuration/response-config-cluster.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "cluster": {
+      "@type": "type.googleapis.com/envoy.api.v2.Cluster",
```

Comment:
this is the wrong response - update to v3

Reply:
hmm - seems like this is the response you get back from the go-control-plane

Reply:
i guess this must be what is configured in `static_resources` - im surprised its not an `api.v3.Cluster`

Reply:
Yeah I would update to v3 if possible. cc @htuch @kyessenov for guidance here.

Reply:
this seems like an issue with go-control-plane - (at least the example i based this on)

Reply:
which is odd because checking here https://github.com/envoyproxy/go-control-plane/tree/master/internal/example i cant see a `v2` anywhere - everything is set to v3

also checking back through this sandbox - all of the config suggests v3  - so im a bit lost as to cause

Reply:
looking at this further its nothing to do with go-control-plane as this is Envoy's configuration for g-c-p

im wondering if there is anything i can change to this config:

```
dynamic_resources:
  ads_config:
    api_type: GRPC
    transport_api_version: V3
    grpc_services:
      - envoy_grpc:
          cluster_name: xds_cluster
  cds_config:
    resource_api_version: V3
    ads: {}
  lds_config:
    resource_api_version: V3
    ads: {}

static_resources:
  clusters:
    - connect_timeout: 1s
      type: strict_dns
      http2_protocol_options: {}
      name: xds_cluster
      load_assignment:
        cluster_name: xds_cluster
        endpoints:
          - lb_endpoints:
              - endpoint:
                  address:
                    socket_address:
                      address: go-control-plane
                      port_value: 18000
```

that would resolve

im happy to open a ticket if what is happening is not expected, or update if it needs something more

Reply:
@mattklein123  lmk if you want me to follow up on this  in some way

Reply:
Ultimately we do want this to return v3 objects, but it's likely that the GCP samples need to be updated. cc @jyotima @kyessenov

Reply:
hmm, i reckon its not gcp thats returning this, its what you get from config above, when you do config_dump in  envoy - its the static part

Reply:
OK not sure what the issue is without digging in more detail. It's possible this is a config dump issue. I'm not sure.

Reply:
let me open a ticket - and we can thrash it out there...

Reply:
@phlax Can you help me with the gcp samples you followed? From the initial pass looks like gcp example uses v3 https://github.com/envoyproxy/go-control-plane/blob/master/internal/example/README.md

Reply:
@jyotimahapatra i think its not a problem with go-control-plane

i opened a ticket here #13817 to raise it